css/main.scss

```diff
@@ -7,7 +7,6 @@ body {
   font-weight: 300;
   font-family: 'Ubuntu', sans-serif;
   letter-spacing: 1px;
-  text-align: justify;
```

Comment:
yeah, not sure about `<li>` - if you say it looks weird we'll leave those left-aligned